### DISCUSSION TITLE
Issue with Protect and unifi-protect-node

### DISCUSSION THREAD

**Discussion by emmav674** [NRCHKB/node-red-contrib-unifi-os#83] [General]
Hello,
I have UniFI UNVR 4.0.6 and Protect 4.1.53. The unifi-protect node has stopped to send ANYTHING but events of motion detection. The only way I see events are sent is enabling motion detection on the camera. To reproduce the issue, just connect a debug node to unifi protect and generate an A.I. event (like person detection). Absolutely no output is produced on debug node. 
If you enable motion detection, than in the debug you can see the output.

<img width="214" alt="Immagine 2024-09-03 193740" src="https://github.com/user-attachments/assets/a0ac8fbf-83c7-408f-9b79-85b011968217">
<img width="379" alt="Immagine 2024-09-03 193130" src="https://github.com/user-attachments/assets/64200926-3ab8-4abe-b06c-780de800231c">
<img width="297" alt="Immagine 2024-09-03 193219" src="https://github.com/user-attachments/assets/3becd97e-9a00-4482-92d3-3cd44f6f0e6d">

**Comment by doctorkb** [comment]
Duplicate of #81

**Comment by gionimi** [comment]
same problem here :(. I updated to version 5.0.34 and there are still no detections. The most stable version for this is 4.0.33, however this version has a bug in recording playback on the protect plataform
